brave/brave-ios
# Fix #1578 - Removes Alamofire

## Summary of Changes

Fixes: https://github.com/brave/brave-ios/issues/1578

Security Review for SSL Pinning: https://github.com/brave/internal/issues/665

## Submitter Checklist:


## Test Plan:
1. Check if referrals are working.
2. Enable charles proxy and intercept ssl requests.
3. Verify that referrals are NOT working with charles proxy enabled.
4. Disable charles proxy and verify that referrals are working again.


## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
